test/test_sparse.py

```diff
@@ -979,15 +979,90 @@ def test_factory_size_check(self):
         indices = self.IndexTensor([[1, 2], [0, 2]])
         values = self.ValueTensor([[1, 1, 1], [1, 1, 1]])
         sizes = torch.Size([3, 3, 2])
-        with self.assertRaisesRegex(RuntimeError, "values and sizes are inconsistent"):
+        with self.assertRaisesRegex(RuntimeError, "values has incorrect size"):
             self.SparseTensor(indices, values, sizes)
 
+    def test_factory_default(self):
+        tensor = self.SparseTensor()
```

Comment:
nit: for new tests, can you use the new constructor?

Reply:
I changed it for all call sites that has arguments. For this one, we don't have the no-arg constructor `torch.sparse_coo_tensor()` yet, but we can probably add one (we don't have `torch.tensor()` though at the moment).